session/session.go

```diff
@@ -160,11 +195,6 @@ func (s *Session) Declassify() *Session {
 	return s
 }
 
-func (s *Session) Refresh(c lifespanProvider) *Session {
-	s.ExpiresAt = time.Now().Add(c.SessionLifespan())
-	return s
-}
-
 func (s *Session) CanBeRefreshed(c refreshWindowProvider) bool {
```

Comment:
In the upstream v0.10.1 tag `CanBeRefreshed` method is defined a few lines below, after the `Refresh` method. We can change it to decrease the difference between our version the the upstream